myapp/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from .views import InputView
+
+urlpatterns = [
+    path("est-sheet-gen/", InputView.as_view(), name="item-create"),
```

Comment:
name="item-create" this is also not a good name for a URL. like which item its being created?